synfig-studio/src/gui/states/state_mirror.cpp

```diff
@@ -71,6 +71,8 @@ enum Axis {
 	AXIS_Y
 } ;
 
+#define GAP (3)
```

Comment:
No need to use `#define`

Just put a `const int gap = 3;` inside the method :)

Reply:
`#define GAP (3)` is used for all the Tools (repeated for each one in their `.cpp` file).
Do you suggest to substitute all those instances?